### ISSUE TITLE
TypeError: _log() got an unexpected keyword argument 'completion'

### ISSUE THREAD

**Issue by pplonski** [TypeFox/monaco-languageclient#305]
I got an error:

```
Error
TypeError: _log() got an unexpected keyword argument 'completion'

Error: TypeError: _log() got an unexpected keyword argument 'completion'
at handleResponse (http://localhost:1212/renderer.dev.js:240484:48)
at processMessageQueue (http://localhost:1212/renderer.dev.js:240299:17)
at http://localhost:1212/renderer.dev.js:240283:13
```

I'm using `python-lsp-server` with Electron and monaco-editor. The code is here https://github.com/pplonski/electron-monaco-python-lsp

I get the error wile write:
```
import numpy
numpy.random.
```

Is this issue from `monaco-languageclient` or from `python-lsp-server`?

Details how to reproduce the problem: https://github.com/python-lsp/python-lsp-server/issues/119

Thank you!

**Comment by rcjsuen** [comment]
Based on the error message and the stack trace you provided in https://github.com/python-lsp/python-lsp-server/issues/119, I think it's safe to say that there's something going wrong with the language server itself.

**Comment by pplonski** [comment]
Fixed by downgrading `jedi` to `0.17.2`. Thank you!
